invenio_rdm_records/resources/serializers/dcat/datacite-to-dcat-ap.xsl

```diff
@@ -1655,7 +1655,10 @@
         <xsl:otherwise>
 <!--
           <rdf:type rdf:resource="{$foaf}Agent"/>
--->
+-->       
+          <xsl:if test="$uri != ''">
+            <dct:identifier rdf:datatype="{$xsd}string"><xsl:value-of select="$nameIdentifier"/></dct:identifier>
```

Comment:
So far this file has been fully synchronized with a repository which is not maintained by us:
https://github.com/ec-jrc/datacite-to-dcat-ap/blob/master/datacite-to-dcat-ap.xsl

We need to decide if we want to:
* Diverge with the upstream version
* Propose a contribution to the upstream version (we recently contributed a bug fix)
* Find another way to achieve the expected result in InvenioRDM.

Reply:
After some discussions, the decision is that we can:
1. Merge this change
2. Propose an PR with the same change upstream on https://github.com/ec-jrc/datacite-to-dcat-ap/